didact.js

```diff
@@ -58,6 +61,23 @@ function updateDom(dom, prevProps, nextProps) {
       )
     })
 
+  const prevStyle = prevProps.style || {}
+  const nextStyle = nextProps.style || {}
+
+  // Remove old styles
+  Object.keys(prevStyle)
+    .filter(isGone(prevStyle, nextStyle))
+    .forEach(name => {
+      dom.style[name] = ""
+    })
+
+  // Set new or changed styles
+  Object.keys(nextStyle)
+    .filter(isNew(prevStyle, nextStyle))
+    .forEach(name => {
+      dom.style[name] = nextStyle[name]
```

Comment:
Looks a little verbose, why not just override style here?

Reply:
Because the style object is read-only: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/style